### ISSUE TITLE
Cant login with normal user with hacking attempt message

### ISSUE THREAD

**Issue by abelmferreira** [nilsteampassnet/TeamPass#2467]
### Steps to reproduce
1. Make a fresh install of 2.1.27.28 and login with admin account and confirm basic url and path settings are corrects

2. Created two folders, two roles, and one normal user

3. After succesufully login with normal user,  3 or 4 seconds later, received a empty information dialog box, same time main.queries.php returns "Hacking attemp..." message, and we are redirect to login page again. With admin users every thing works great!

![image](https://user-images.githubusercontent.com/2151109/48104755-dfb55f00-e21b-11e8-876c-3bdfe23f1abe.png)

![image](https://user-images.githubusercontent.com/2151109/48104776-f52a8900-e21b-11e8-8ffb-4f5f6cb06005.png)

### Expected behaviour
No empty dialog box message and don't close login session.

### Actual behaviour
Closed session and forced back to login page some seconds after

### Server configuration
**Operating system:**
Ubuntu 18.04

**Web server:**
Apache 2.4.29

**Database:**
Mysql 5.7.24

**PHP version:**
PHP 7.2.10

**Teampass version:**
Teampass 2.1.27.28

**Teampass configuration file:**
```
<?php
global $SETTINGS;
$SETTINGS = array (
    'max_latest_items' => '10',
    'enable_favourites' => '1',
    'show_last_items' => '1',
    'enable_pf_feature' => '0',
    'log_connections' => '0',
    'log_accessed' => '1',
    'time_format' => 'H:i:s',
    'date_format' => 'd/m/Y',
    'duplicate_folder' => '0',
    'item_duplicate_in_same_folder' => '0',
    'duplicate_item' => '0',
    'number_of_used_pw' => '3',
    'manager_edit' => '1',
    'cpassman_dir' => '/home/me/web/teampass.domain.com.br/public_html',
    'cpassman_url' => 'https://teampass.domain.com.br',
    'favicon' => 'https://teampass.domain.com.br/favicon.ico',
    'path_to_upload_folder' => '/home/me/web/teampass.domain.com.br/public_html/upload',
    'url_to_upload_folder' => 'https://teampass.domain.com.br/upload',
    'path_to_files_folder' => '/home/me/web/teampass.domain.com.br/public_html/files',
    'url_to_files_folder' => 'https://teampass.domain.com.br/files',
    'activate_expiration' => '0',
    'pw_life_duration' => '0',
    'maintenance_mode' => '0',
    'enable_sts' => '0',
    'encryptClientServer' => '1',
    'cpassman_version' => '2.1.27',
    'ldap_mode' => '0',
    'ldap_type' => '0',
    'ldap_suffix' => '0',
    'ldap_domain_dn' => '0',
    'ldap_domain_controler' => '0',
    'ldap_user_attribute' => '0',
    'ldap_ssl' => '0',
    'ldap_tls' => '0',
    'ldap_elusers' => '0',
    'ldap_search_base' => '0',
    'ldap_port' => '389',
    'richtext' => '0',
    'allow_print' => '0',
    'roles_allowed_to_print' => '0',
    'show_description' => '1',
    'anyone_can_modify' => '0',
    'anyone_can_modify_bydefault' => '0',
    'nb_bad_authentication' => '0',
    'utf8_enabled' => '1',
    'restricted_to' => '0',
    'restricted_to_roles' => '0',
    'enable_send_email_on_user_login' => '0',
    'enable_user_can_create_folders' => '0',
    'insert_manual_entry_item_history' => '0',
    'enable_kb' => '0',
    'enable_email_notification_on_item_shown' => '0',
    'enable_email_notification_on_user_pw_change' => '0',
    'custom_logo' => '',
    'custom_login_text' => '',
    'default_language' => 'english',
    'send_stats' => '0',
    'send_statistics_items' => 'stat_country;stat_users;stat_items;stat_items_shared;stat_folders;stat_folders_shared;stat_admins;stat_managers;stat_ro;stat_mysqlversion;stat_phpversion;stat_teampassversion;stat_languages;stat_kb;stat_suggestion;stat_customfields;stat_api;stat_2fa;stat_agses;stat_duo;stat_ldap;stat_syslog;stat_stricthttps;stat_fav;stat_pf;',
    'send_stats_time' => '1538945256',
    'get_tp_info' => '1',
    'send_mail_on_user_login' => '0',
    'sending_emails' => '0',
    'nb_items_by_query' => 'auto',
    'enable_delete_after_consultation' => '0',
    'enable_personal_saltkey_cookie' => '0',
    'personal_saltkey_cookie_duration' => '31',
    'email_smtp_server' => '',
    'email_smtp_auth' => '',
    'email_auth_username' => '',
    'email_auth_pwd' => '',
    'email_port' => '',
    'email_security' => '',
    'email_server_url' => '',
    'email_from' => '',
    'email_from_name' => '',
    'pwd_maximum_length' => '40',
    'google_authentication' => '0',
    'delay_item_edition' => '0',
    'allow_import' => '0',
    'proxy_ip' => '',
    'proxy_port' => '',
    'upload_maxfilesize' => '10mb',
    'upload_docext' => 'doc,docx,dotx,xls,xlsx,xltx,rtf,csv,txt,pdf,ppt,pptx,pot,dotx,xltx',
    'upload_imagesext' => 'jpg,jpeg,gif,png',
    'upload_pkgext' => '7z,rar,tar,zip',
    'upload_otherext' => 'sql,xml',
    'upload_imageresize_options' => '1',
    'upload_imageresize_width' => '800',
    'upload_imageresize_height' => '600',
    'upload_imageresize_quality' => '90',
    'use_md5_password_as_salt' => '0',
    'ga_website_name' => 'TeamPass for ChangeMe',
    'api' => '0',
    'subfolder_rights_as_parent' => '0',
    'show_only_accessible_folders' => '0',
    'enable_suggestion' => '0',
    'otv_expiration_period' => '7',
    'default_session_expiration_time' => '60',
    'duo' => '0',
    'enable_server_password_change' => '0',
    'ldap_object_class' => '0',
    'bck_script_path' => '/home/me/web/teampass.domain.com.br/public_html/backups',
    'bck_script_filename' => 'bck_teampass',
    'syslog_enable' => '0',
    'syslog_host' => 'localhost',
    'syslog_port' => '514',
    'manager_move_item' => '0',
    'create_item_without_password' => '0',
    'otv_is_enabled' => '0',
    'agses_authentication_enabled' => '0',
    'item_extra_fields' => '0',
    'saltkey_ante_2127' => 'none',
    'migration_to_2127' => 'done',
    'files_with_defuse' => 'done',
    'timezone' => 'UTC',
    'enable_attachment_encryption' => '1',
    'personal_saltkey_security_level' => '50',
    'ldap_new_user_is_administrated_by' => '0',
    'disable_show_forgot_pwd_link' => '0',
    'offline_key_level' => '0',
    'enable_http_request_login' => '0',
    'ldap_and_local_authentication' => '0',
    'secure_display_image' => '1',
    'upload_zero_byte_file' => '0',
    'upload_all_extensions_file' => '0',
    'bck_script_passkey' => 'HLjvRn4k4twxh1sGDASBT99NMYZsRb2EsCNJd4zC',
    'admin_2fa_required' => '1',
);
```

**Updated from an older Teampass or fresh install:**
Fresh Install

### Client configuration
Chorme 67 on Mac OSX
Chorme 70 on Windows 10
Safari 12.0.1 and Firefox 63.0.1 on Mac OSX

**Operating system:**
Mac OSX High Sierra 10.13.6 and Windows 10

#### Web server error log
```
No errors
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
No errors
```

**Comment by nilsteampassnet** [comment]
Duplicate of #2465

Anyhow thank you for the details.

Can you tell me if in table `teampass_users`, you have a field called `user_ip_lastdate`?

**Comment by nilsteampassnet** [comment]
As I cannot reproduce this, I've performed a code review based upon your details.
I did a change inside the upgrade process and committed it in `development` branch.
Can you download it and test if you face the same issue?
Thanks

**Comment by gertfriend** [comment]
I tried the development version of upgrade_run_2.1.27.php but for me the issue persists.

**Comment by abelmferreira** [comment]
> Can you tell me if in table `teampass_users`, you have a field called `user_ip_lastdate`?

Yes, exist with null values, also when user is logged in

**Comment by abelmferreira** [comment]
> I did a change inside the upgrade process and committed it in `development` branch.
> Can you download it and test if you face the same issue?

I clone the repo, change to development branch and do de normal upgrade process coping files overwrite existing one, and then runing /install/upgrade.php

All steps successfully, but nothing diferent, same empty information box and hackin attemp message reply from main.query.php

**Comment by nilsteampassnet** [comment]
Fixed in [2.1.27.29](https://github.com/nilsteampassnet/TeamPass/releases/tag/2.1.27.29)

Just overwrite the files. No DB change.

**Comment by abelmferreira** [comment]
Thanks!! Fixed!
